src/transaction/log_impl.h

```diff
@@ -926,7 +926,7 @@ extern int logtb_find_client_name_host_pid (int tran_index, const char **client_
 					    const char **client_user_name, const char **client_host_name,
 					    int *client_pid);
 #if !defined(NDEBUG)
-extern void logpb_debug_check_log_page (THREAD_ENTRY * thread_p, void *log_pgptr_ptr);
+extern void logpb_debug_check_log_page (THREAD_ENTRY * thread_p, const void *log_pgptr_ptr);
```

Comment:
Imo, it is more clear if the argument is directly a const pointer to LOG_PAGE.
```suggestion
extern void logpb_debug_check_log_page (THREAD_ENTRY * thread_p, const LOG_PAGE *log_pgptr_ptr);
```

Reply:
Done, but some casts have moved elsewhere.